home-page.html

```diff
@@ -0,0 +1,168 @@
+                        </ul>
+                    </div>
+                </li>
+                <li class="nav-branch">
+                    <div>
+                        <a href="" class="nav-links">INTERNAL PARTS</a>
+                        <ul class="sub-navbar">
+                            <li><a href="" class="sub-nav-links">Gearboxes</a></li>
+                            <li><a href="" class="sub-nav-links">Motors</a></li>
+                        </ul>
+                    </div>
+                </li>
+            </ul>
+        </div>
+
+        <div class="home-page-content">
+            <div class="home-product">
+                <div class="product-body">
+                    <img class="img-product" src="images/product/ak101-img.jpeg" alt=" an ak101 ">
+                    <hr>
+                    <h5 class="product-title">
+                        AK-101 
+                    </h5>
+                    <div class="description-container">
+                        <p class="product-description">very good rifle i'm speechless </p>
+                    </div>
+                    <button class="product-add-button">ADD</button>
+                    <button class="product-plus">+</button>
+                    <p class="product-number">1</p>
+                    <button class="product-minus">-</button>
+                </div>
+            </div>
+        </div>
+
+        <div class="home-footer">
+            <h2 class="footer-title">TEST</h2>
+            <div class="footer-contact">
+                <p class="contact-text">INFO INFO</p>
+                <p class="contact-text">ALTRA INFO</p>    
+            </div>
+            <div class="footer-img">
+                <a href=""><img class="social-img" src="images/home-page/instagram-img.png" alt=" the instagram logo"></a>
+                <a href=""><img class="social-img" src="images/home-page/facebook-img.png" alt=" the facebook logo"></a>
+                <a href=""><img class="social-img" src="images/home-page/whatsapp-img.png" alt=" the whatsapp logo"></a>
+                <a href=""><img class="social-img" src="images/home-page/x-img.png" alt=" the twitter logo"></a>
+            </div>
+            
+        </div>
+
+        <script type="text/javascript">
+            window.onload = function (){
+                
+                let itemCount = 1;
+                let buttonMinus;
+                let buttonPlus;
+
+               // increase and decrease methods for the product.
+
+                buttonMinus = document.querySelector(".product-minus");
+                buttonPlus = document.querySelector(".product-plus");
+               
+                buttonMinus.addEventListener("click", decrement);
+                buttonPlus.addEventListener("click", increment);
+
+                function increment(){
+                   let incrementItemCount = document.querySelector(".product-number");
```

Comment:
1) This is not a let! It's a const.

2) Also you don't even need to save the reference if you handle it like that

3) The best solution is to save the reference at the start of the onload and just edit the inner count.

Reply:
fixed